apps/core-angular-js/package.json

```diff
@@ -20,8 +20,8 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@clr/city": "^1.1.0",
-    "@clr/core": "./packages/core/dist/core",
+    "@cds/city": "^2.0.0",
```

Comment:
Is 2.0 correct here?

Reply:
no, Clarity City is going to stay `1.1.0`. I'll push an update today.

Reply:
I pushed the change and will watch the build.